### PR TITLE
refactor(core-p2p): remove chunk broadcasting

### PR DESCRIPTION
**Pull request by mvega3**
## Proposed changes
This is a first implementation of the p2p algorithm specified by @zillionn in #1494. I'm not familiar enough with the rationale behind the limit to suggest which solution is best -- the current code, Zillion's first solution (which is what this PR attempts to implement), or his second solution: to forgo rebroadcasting.

If the decision is to forgo rebroadcasting, most of this code can be rolled back and replaced with the previous error code commented out in this PR.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)

## Checklist
- [x] I have read the [CONTRIBUTING](https://docs.ark.io/developers/guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kristjank** [comment]
personally - I recommend removing this and hardcoding the maxRequestValue. We will do more harm otherwise.

**Comment by mvega3** [comment]
@kristjank noted, I'll make those changes

File: packages/core-p2p/lib/peer.js [hunk 1]
```diff
@@ -300,15 +291,23 @@ module.exports = class Peer {
...
-      },
-      {
-        headers: this.headers,
-        timeout: 8000,
-      },
-    )
+    let response
+
+    try {
+      response = await this.__post(
```
Comment (faustbrian) [inline packages/core-p2p/lib/peer.js]: We can just return here, there is no point in assigning it to a variable if we are not going to do anything with it afterwards besides returning it.

File: packages/core-p2p/lib/peer.js [hunk 2]
```diff
@@ -76,16 +76,7 @@ module.exports = class Peer {
     try {
```
Comment (faustbrian) [inline packages/core-p2p/lib/peer.js]: This try/catch won't do anything anymore because __broadcastTransactions already catches any exceptions.

**Pull request merged**